gau2grid/c_wrapper.py

```diff
@@ -19,7 +19,7 @@
 # First check the local folder
 try:
     abs_path = os.path.dirname(os.path.abspath(__file__))
-    cgg = np.ctypeslib.load_library("gg", abs_path)
+    cgg = np.ctypeslib.load_library("libgg", abs_path)
```

Comment:
Question for you, @loriab.  I had to make this change here for macOS (simply making a symlink called `gg` didn't work).  Do you know if this is a good idea?  No problem if not - I can continue down the ctypes documentation rabbithole I dived down yesterday.

Reply:
Well, I think I have the answer - Travis didn't like the change.  Will revert it and figure out a different strategy for macOS....

Reply:
I'm not sure what's going on with `load_library`. DGAS [just did the opposite](https://github.com/dgasmith/gau2grid/commit/bdd46992654239f5eb08b9db72052779ebf6d17c) of what you propose a few weeks ago. And according to the docs, either should work. Plus, I can't track why it needed to change in the July case. I know in the c-f build, Mac likes the current master.

Sorry, that's all pretty useless. One point of info I can add is that the linux lib with SONAME set is renamed when it's copied to the python pkg directory and that happens https://github.com/dgasmith/gau2grid/blob/master/CMakeLists.txt#L159.

Reply:
Thanks for that info - very helpful.  I have no clue what I'm doing, but reverting the change (to match current master) fixes the travis build.  If I do that on my laptop (running Mojave), the library fails to load and most of the pytests are skipped.  Very strange, but at least I know a workaround for local development - just changing the `gg` to `libgg` does the trick.

Reply:
Maybe something stupid?

```python
cgg = np.ctypeslib.load_library("gg", abs_path)
if cgg is None:
    cgg = np.ctypeslib.load_library("libgg", abs_path)
```

Reply:
maybe make a note of your current numpy version, then update numpy and see if that's happier with the plain "gg"

Reply:
I suspect my numpy version 1.16 could be at fault, but I hate the idea that the code is version dependent.  I implemented something similar to DGAS's suggestion - the function throws if it doesn't find the right library, so I just threw in a nested `try/except` block.